tests/unit/libbpf_test.cpp

```diff
@@ -2323,10 +2320,10 @@ TEST_CASE("libbpf_prog_type_by_name_test", "[libbpf]")
     bpf_prog_type prog_type;
     bpf_attach_type expected_attach_type;
 
-    // Try a cross-platform type.
-    REQUIRE(libbpf_prog_type_by_name("xdp", &prog_type, &expected_attach_type) == 0);
-    REQUIRE(prog_type == BPF_PROG_TYPE_XDP);
-    REQUIRE(expected_attach_type == BPF_XDP);
+    // // Try a cross-platform type.
+    // REQUIRE(libbpf_prog_type_by_name("xdp", &prog_type, &expected_attach_type) == 0);
+    // REQUIRE(prog_type == BPF_PROG_TYPE_XDP_TEST);
+    // REQUIRE(expected_attach_type == BPF_XDP_TEST);
```

Comment:
More commented out code to fix or remove.

Reply:
Thanks for catching this - I commented out some tests during development and looks like I forgot to uncomment some things. Will check this file carefully.